WordPress/Classes/ViewRelated/Reader/ReaderTopicsCardCell.swift

```diff
@@ -2,36 +2,126 @@ import UIKit
 
 /// A cell that displays topics the user might like
 ///
-class ReaderTopicsCardCell: ReaderTopicsTableCardCell {
-    private let cellIdentifier = "TopicCell"
+class ReaderTopicsCardCell: UITableViewCell, NibLoadable {
```

Comment:
```suggestion
class ReaderTopicsCardCell: UITableViewCell, NibReusable {
```

Reply:
Done in: 616998ecec24851a03b947bb70a219a03766ce49